.circleci/config.yml

```diff
@@ -51,6 +51,49 @@ common-steps:
       paths:
         - /caches/layers.tar
 
+  - &focalcreatecachedir
```

Comment:
we can probably re-use the same cache dir as for Xenial just specifying different layer files